### PR TITLE
chore: support specta 2

### PR DESCRIPTION
**Pull request by brainbicycle**
👋 
- updates the podspec to support latest version of Specta
- changes to xcodeproj due to later cocoapods version

The tests in the example project are passing with these changes but let me know if any other tests or changes are needed!
Thanks for the library!

### CODE REVIEW

File: FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -213,7 +213,6 @@
 				A24E011B17F9A42A001252DB /* Frameworks */,
 				A24E011C17F9A42A001252DB /* Resources */,
 				279D0DCE5C316DE54BD5FF49 /* [CP] Embed Pods Frameworks */,
-				F04559B5D97B33B48486C223 /* [CP] Copy Pods Resources */,
```
Comment (brainbicycle) [inline FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo.xcodeproj/project.pbxproj:216]: believe most of these changes are due to cocoapods version update + updating minimum deployment target but let me know if needs cleanup

File: FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDemoSpecs.m [hunk 2]
```diff
@@ -282,40 +282,6 @@
...
-            }
-            
-            expect(view).toNot.recordSnapshot();
-            expect(view).to.haveValidSnapshot();
-            
-//            [view dealloc]; // should be called manually
-        }
-        
-        free(ptr);
-    });
```
Comment (brainbicycle) [inline FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDemoSpecs.m:318]: this test was crashing with EXC_BAD_ACCESS when I ran, happy to try to debug further if you all want

File: FBSnapshotTestCaseDemo/Podfile [hunk 3]
```diff
@@ -12,4 +12,12 @@ post_install do |installer|
       config.build_settings['SWIFT_VERSION'] = 4.0
     end
   end
+
+  installer.generated_projects.each do |project|
+      project.targets.each do |target|
+          target.build_configurations.each do |config|
+              config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '14.0'
```
Comment (brainbicycle) [inline FBSnapshotTestCaseDemo/Podfile:19]: as of this: https://github.com/CocoaPods/CocoaPods/issues/4859 pods don't automatically inherit the minimum deployment target, Specta fails to build with missing frameworks if too low but probably could be lower than 14

File: Gemfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods', '1.1.1'
+gem 'cocoapods', '1.15.2'
```
Comment (brainbicycle) [inline Gemfile:3]: install failures with older versions of cocoapods

**Comment by dblock** [comment]
I love it! Haven't touched this project in a long time, but would be glad to revive it (and welcome a new maintainer :)). Let's replace Travis-CI by GHA first to get CI/CD working again? Want to PR that separately?

**Comment by brainbicycle** [comment]
> I love it! Haven't touched this project in a long time, but would be glad to revive it (and welcome a new maintainer :)). Let's replace Travis-CI by GHA first to get CI/CD working again? Want to PR that separately?

hey! sure I can take a stab at that, can't promise too much bandwidth but happy to help where I can.

**Comment by dblock** [comment]
> > I love it! Haven't touched this project in a long time, but would be glad to revive it (and welcome a new maintainer :)). Let's replace Travis-CI by GHA first to get CI/CD working again? Want to PR that separately?
> 
> hey! sure I can take a stab at that, can't promise too much bandwidth but happy to help where I can.

Great. LMK if you are *not* working on (re)adding CI and I can try to find time.

**Comment by dblock** [comment]
Merged #49, want to rebase this one?

**Review by dblock — APPROVED** [review]

**Comment by dblock** [comment]
@brainbicycle Thanks! Want to come join as a maintainer of this project? I haven't worked on it in years and I'm glad someone is using it.

**Comment by brainbicycle** [comment]
> @brainbicycle Thanks! Want to come join as a maintainer of this project? I haven't worked on it in years and I'm glad someone is using it.

hey sure, thanks for asking! Will help where I can.

**Comment by dblock** [comment]
Added, https://github.com/dblock/ios-snapshot-test-case-expecta/invitations

**Pull request merged**